daemons/gptp/common/avbts_osipc.hpp

```diff
@@ -85,7 +85,8 @@ class OS_IPC {
 		uint32_t sync_count,
 		uint32_t pdelay_count,
 		PortState port_state,
-		bool asCapable ) = 0;
+		bool asCapable,
+      		uint64_t mean_path_delay ) = 0;
```

Comment:
Maybe I'm looking at the wrong commit but it looks like there are still some alignment issues.

Reply:
I think I got this one and the other alignment comment, and I didn't see any other alignment problems from a quick review. Let me know if we need any other changes before we commit this and the perforce files.